Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -710,7 +711,7 @@ public extension String {
 	/// - Parameter separator: separator to split string by.
 	/// - Returns: array of strings separated by given string.
 	public func splitted(by separator: Character) -> [String] {
```

Comment:
Not sure about this one but it looks like `split(separator: Element)` is the way to go